Update Readme and add extended Guide for new developers

niweb opened the pull request.

LocoVIB commented:
Good guide for new team members!
Do we really want to make the links to trello and figma public available?

niweb commented:
Yes, you're right, we probably shouldn't. I removed the direct links and replaced them with general links to the services. (Also force-pushed, so that it won't appear in the history.)
